tests/app/broadcast_message/test_rest.py

```diff
@@ -151,7 +158,7 @@ def test_update_broadcast_message_allows_edit_while_not_yet_live(admin_request,
     )
 
     assert response['starts_at'] == '2020-06-01T20:00:01.000000Z'
-    assert response['areas'] == ['london', 'glasgow']
+    assert response['areas'] == ['london']
```

Comment:
We appear to be updating the broadcast and setting the areas to london and glasgow but the response shows just london. How is this possible? Does this mean the areas aren't being updated for some reason? Should we be updating `_data={'starts_at': '2020-06-01 20:00:01', 'areas': ['london', 'glasgow']},` to also include `simple_polygons`?

Reply:
good catch!

Reply:
I fixed that test now, and I asserted that simple_polygons have updated correctly :).